### PR TITLE
Upgrade to latest Rails 3

### PR DESCRIPTION
**Pull request by calumgunn**
Use 3.2.22, upgrading from 3.2.18.

### CODE REVIEW

**Comment by bensymonds** [comment]
It's upgrading more than rails, but I'm :+1: with it all. It makes sense to freshen everything up as much as possible before we cut v2.

**Comment by jgwhite** [comment]
Hold up, this is gem dependency, right? Should it be `~> 3.2.0`? Other Slices won’t work with Rails `3.2.0 -> v3.2.21`. Am I missing something?

**Comment by bensymonds** [comment]
Oops yeah @jgwhite is right. It's ok to update the `Gemfile.lock`, but the only effect this has is to change the dependencies that get installed when you're _developing_ Slices. When someone installs Slices in their project, _they_ determine the versions of all those dependencies with _their_ Gemfile - within the constraints Slices provides in its gemspec. So if we change the gemspec like this, we're forcing people to use 3.2.22 unnecessarily (Slices would still work with 3.2.18). And actually I'm not even sure why it's 3.2.18 - is there something specific in 3.2.18 that Slices _needs_? Anyway I think we can leave it at that now - I doubt many people are affected by it. If they are they should probably upgrade Rails anyway :sunglasses: 

So basically we can keep the `Gemfile.lock` changes and just make this PR be a refresh of gem versions for development. And if we're doing that, we may as well do a full `bundle update` - this is what CI does each time, to make sure Slices is working with the absolute latest versions of all its dependencies (and dependencies' dependencies...). (Possibly you need to delete `Gemfile.lock` before doing `bundle update`. I'm not 100% sure that everything gets updated properly without doing that.)

**Comment by calumgunn** [comment]
I'm gonna open a new PR with a suitable branch name!
